src/java/com/zimbra/graphql/models/inputs/GQLSearchRequestInput.java

```diff
@@ -72,586 +75,275 @@
-     *
-     * @param calItemExpandStart the calItemExpandStart value to set
-     */
-    @GraphQLInputField(name = "calItemExpandStart", description="Start time in milliseconds for the range to `includeMemberOf` instances for calendar items. If `calExpandInstStart` and `calExpandInstEnd` are specified, and the search types `includeMemberOf` calendar item types (e.g. appointment), then the search results `includeMemberOf` the instances for calendar items within that range in the form described in the description of the response. ***IMPORTANT NOTE: Calendar Items that have no instances within that range are COMPLETELY EXCLUDED from the results. Calendar Items with no data (such as Tasks with no date specified) are included, but with no instance information***")
+    @GraphQLInputField(name = GqlConstants.CALENDAR_ITEM_EXPANDED_START, description="Start time in milliseconds for the range to `includeMemberOf` instances for calendar items. If `calExpandInstStart` and `calExpandInstEnd` are specified, and the search types `includeMemberOf` calendar item types (e.g. appointment), then the search results `includeMemberOf` the instances for calendar items within that range in the form described in the description of the response. ***IMPORTANT NOTE: Calendar Items that have no instances within that range are COMPLETELY EXCLUDED from the results. Calendar Items with no data (such as Tasks with no date specified) are included, but with no instance information***")
     public void setCalItemExpandStart(Long calItemExpandStart) {
         this.calItemExpandStart = calItemExpandStart;
     }
 
-    /**
-     * Gets the value of calItemExpandEnd.
-     *
-     * @return the calItemExpandEnd value
-     */
     public Long getCalItemExpandEnd() {
         return calItemExpandEnd;
     }
 
-    /**
-     * Time in milliseconds to end the span of time started by `calItemExpandStart` for calendar items from.
-     *
-     * @param calItemExpandEnd The calItemExpandEnd value to set
-     */
-    @GraphQLInputField(name = "calItemExpandEnd", description="End time in milliseconds for the range to `includeMemberOf` instances for calendar items from.")
+    @GraphQLInputField(name = GqlConstants.CALENDAR_ITEM_EXPANDED_END, description="End time in milliseconds for the range to `includeMemberOf` instances for calendar items from.")
     public void setCalItemExpandEnd(Long calItemExpandEnd) {
         this.calItemExpandEnd = calItemExpandEnd;
     }
 
-    /**
-     * Gets the value of query.
-     *
-     * @return The query value
-     */
     @GraphQLNonNull
     public String getQuery() {
         return query;
     }
 
-    /**
-     * Query string to use for the search.
-     *
-     * @param query The query string to use
-     */
-    @GraphQLInputField(name = "query", description="Query string to use for the search")
+    @GraphQLInputField(name = GqlConstants.QUERY, description="Query string to use for the search")
     public void setQuery(String query) {
```

Comment:
The non-null needs to go here not on the getter. `public void setQuery(@GraphQLNonNull String query) {`

Reply:
Review this potential issue in other places. Ensure all non-nulls exist for this class and others.